pom.xml

```
@@ -66,6 +66,7 @@ under the License.
     <properties>
         <flink.version>1.13.1</flink.version>
         <debezium.version>1.5.2.Final</debezium.version>
+        <geometry.version>2.0.0</geometry.version>
```

```suggestion
        <geometry.version>2.2.0</geometry.version>
```
`flink-table-planner` also contains this dependency,  could we use same version ?